AzureCalling/Controllers/JoinCallViewController.swift

```diff
@@ -166,7 +167,12 @@ class JoinCallViewController: UIViewController {
             return
         }
         let callConfig = JoinCallConfig(joinId: joinId, displayName: displayName ?? "", callType: joinCallType)
-        callingContext.startCallComposite(callConfig)
+        busyOverlay.presentIn(view: view)
+        self.callingContext.startCallComposite(callConfig, completionHandler: { [weak self] in
+            DispatchQueue.main.async {
```

Comment:
Same here, navigateToCall is called from the main thread.
Generally, we don't want to async dispatch unless needed.